src/main/java/com/scalar/database/storage/cassandra/TableMetadata.java

```diff
@@ -0,0 +1,46 @@
+package com.scalar.database.storage.cassandra;
+
+import com.datastax.driver.core.ColumnMetadata;
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.annotation.concurrent.ThreadSafe;
+
+@ThreadSafe
+public class TableMetadata {
+  private final ClusterManager manager;
+  private final String keyspace;
+  private final String tableName;
+  private final com.datastax.driver.core.TableMetadata tableMetadata;
+  private final Set<String> partitionKeyNames;
+  private final Set<String> clusteringColumnNames;
+
+  public TableMetadata(ClusterManager manager, String keyspace, String tableName) {
+    this.manager = manager;
```

Comment:
It might be not necessary to use properties to preserve manager, keyspace and tableName individually since they seem only used to get Cassandra's table metadata.

Reply:
Good catch ! Right. Probably Passing `com.datastax.driver.core.TableMetadata` only to the constructor might be even better.
Let me work on that.

Reply:
@scalarindetail I fixed as below. Please take an another look !
https://github.com/scalar-labs/scalardb/pull/28/commits/42b0151d6e656efc838b15df5f1dd45150deee88

Reply:
LGTM